hack/overlays/set-envoy-daemonset-resources-limits.yaml

```
@@ -0,0 +1,28 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.all,expects="1+"
+#@overlay/match-child-defaults when=1
+---
+spec:
+  template:
+    spec:
+      containers:
+        #@overlay/match by="name"
+        - name: shutdown-manager
+          resources:
+            requests:
+              cpu: 40m
+              memory: 40Mi
+            limits:
+              cpu: 400m
+              memory: 400Mi
+        #@overlay/match by="name"
+        - name: envoy
+          resources:
```

We'll want want the same annotation as above

```suggestion
          #@overlay/match missing_ok=True
          resources:
```